docs/operator-manual/applicationset/Generators-Pull-Request.md

```diff
@@ -351,15 +351,18 @@ spec:
   generators:
   - pullRequest:
       # ...
-      # Include any pull request ending with "argocd". (optional)
+      # Include any pull request branch ending with "argocd" 
+      # and pull request title starting with "feat:". (optional)
       filters:
       - branchMatch: ".*-argocd"
+      - titleMatch: "^feat:"
```

Comment:
Could you also add this example to docs/operator-manual/applicationset.yaml?

Reply:
done, thanks!!